apps/src/codebridge/ControlButtons/index.tsx

```diff
@@ -87,13 +91,24 @@ const ControlButtons: React.FunctionComponent = () => {
 
   const handleRun = (runTests: boolean) => {
     if (onRun) {
-      onRun(runTests, dispatch, source);
+      setIsRunning(true);
```

Comment:
I was curious why the "Run" button feels a very tiny bit laggy (it doesn't disappear quickly after click)? I don't think you should change this for this PR fwiw. I assumed that run was being gated on a slow operation that had to start, but this looks like a pretty instant state change. 🤷 Is react rendering taking that long to kick in? I wouldn't have even noticed the lag except the UX is so slick in general, the fading buttons etc are sweet.